### DISCUSSION TITLE
Welcome to discussions

### DISCUSSION THREAD

**Discussion by 0u44** [CPScript/Kitty-Tools#40] [Announcements]
Talk and ask questions here

**Comment by zmdtxldd** [comment]
![Screenshot_20240405_101505.jpg](https://github.com/CPScript/Kitty-Tools/assets/143540972/7f0f6711-ac3c-47b4-a47d-1b98d0619c26)

please give me example how do I have to type ID correctly

**Comment by 0u44** [comment]
you need to type the quiz is part (random bunch of numbers at the end) and not the actual link

**Comment by flauts** [comment]
what the sigma

**Comment by Swagxx572k** [comment]
Can we get a version that works on iPad
